examples/lab-robust/v2/assessments/step_one_check.rb

```diff
@@ -0,0 +1,15 @@
+def check(handles:, maximum_score:, resources:)
```

Comment:
```suggestion
def steo_one_check(handles:, maximum_score:, resources:)
```

Reply:
does this one actually matter? I don't think method name has to match, Gw just copies content of this file as is

Reply:
The name should match the `method_name:` in the assessments section. I think V2 activity tracking also picks up the name `check()` in certain cases.

Reply:
done